client/src/components/Dashboard/index.js

```diff
@@ -6,17 +6,153 @@ import {
 import LiveTicker from "./LiveTicker";
 import YourFavorites from "./YourFavorites";
 import KeyIndicators from "./KeyIndicators";
 import styles from "../Styles/Dashboard/styles";
 import GridContainer from "../Styles/Dashboard/GridContainer.jsx";
+import deburr from "lodash/deburr";
+import Autosuggest from "react-autosuggest";
+import match from "autosuggest-highlight/match";
+import parse from "autosuggest-highlight/parse";
+import { suggestions } from "../Reports/suggestions";
+import axios from "axios";
+import firebase from "firebase";
+
+const renderInputComponent = inputProps => {
+  const { classes, inputRef = () => { }, ref, ...other } = inputProps;
+
+  return (
+    <TextField
+      fullWidth
+      InputProps={{
+        inputRef: node => {
+          ref(node);
+          inputRef(node);
+        },
+        classes: {
+          input: classes.input
+        }
+      }}
+      {...other}
+    />
+  );
+};
+
+const renderSuggestion = (suggestion, { query, isHighlighted }) => {
+  const matches = match(suggestion.label, query);
+  const parts = parse(suggestion.label, matches);
+
+  return (
+    <MenuItem selected={isHighlighted} component="div">
+      <div>
+        {parts.map((part, index) =>
+          part.highlight ? (
+            <span key={String(index)} style={{ fontWeight: 500 }}>
+              {part.text}
+            </span>
+          ) : (
+              <strong key={String(index)} style={{ fontWeight: 300 }}>
+                {part.text}
+              </strong>
+            )
+        )}
+      </div>
+    </MenuItem>
+  );
+};
+
+const getSuggestions = value => {
+  const inputValue = deburr(value.trim()).toLowerCase();
+  const inputLength = inputValue.length;
+  let count = 0;
+
+  return inputLength === 0
+    ? []
+    : suggestions.filter(suggestion => {
+      const keep =
+        count < 5 &&
+        suggestion.label.slice(0, inputLength).toLowerCase() === inputValue;
+
+      if (keep) {
+        count += 1;
+      }
+
+      return keep;
+    });
+};
+
+const getSuggestionValue = suggestion => {
+  return suggestion.label;
+};
+
+
 class Dashboard extends Component {
+  state = {
+    value: 0,
+    data: [],
+    ticker: "",
+    single: "",
+    popper: "",
+    suggestions: [],
+    uid: firebase.auth().currentUser.uid,
+  };
+
+  handleSuggestionsFetchRequested = ({ value }) => {
+    this.setState({
+      suggestions: getSuggestions(value)
+    });
+  };
+
+  handleSuggestionsClearRequested = () => {
+    this.setState({
+      suggestions: []
+    });
+  };
+
+  handleChange = name => (event, { newValue }) => {
+    this.setState({
+      [name]: newValue
+    });
+  };
+
+  onSuggestionSelected = (
+    event,
+    { suggestion, suggestionValue, suggestionIndex, sectionIndex, method }
+  ) => {
+    const newSymbol = {
+      symbol: suggestionValue,
+      uid: this.state.uid
+    }
+    axios.post('https://pickemm.herokuapp.com/api/favorites', newSymbol)
```

Comment:
I kind of think that the selected stock should just be added to the cards on top. Then, the user can choose to favorite at some point if they want to. There's no indication when you're typing in that search box that it's going to be added to Favorites.

Reply:
those are hard coded. there is no list in the db to add them to.